### PR TITLE
[insights-agent] fleet install mechanism

### PR DESCRIPTION
**Pull request by rbren**
**Why This PR?**
_a short description of why this PR is needed_

This add support for installing Insights across many clusters with a single admin token. This will obviate the need to log into the UI, add a cluster, and copy the access token for every cluster in your fleet.

Specifically, you can run:
```bash
helm upgrade --install insights-agent fairwinds-stable/insights-agent \
  --namespace insights-agent \
  --set polaris.enabled="true" \
  --set insights.organization="acme-co"  \
  --set insights.cluster="your-name-for-this-cluster" \
  --set insights.adminToken="thisisanadmintoken" \
  --set fleetInstall=true \
  --set insights.tokenSecretName=insights-agent-token
```

This will create the cluster if it doesn't already exist, or get the current token if it does.

On future runs, you can omit the admin token and `fleetInstall`, and simply run
```bash
helm upgrade --install insights-agent fairwinds-stable/insights-agent \
  --namespace insights-agent \
  --set polaris.enabled="true" \
  --set insights.organization="acme-co"  \
  --set insights.cluster="your-name-for-this-cluster" \
  --set insights.tokenSecretName=insights-agent-token
```

Fixes #

**Changes**
Changes proposed in this pull request:

*
*

**Checklist:**

* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] Any new values have been added to the README for the Chart, or helm-docs has been run for the charts that support it.

### CODE REVIEW

**Review by baderbuddy — APPROVED** [review]
This is good, but could we change `adminToken` to `apiToken` or something? I'd really like to create a special bot role for this in the future.

File: stable/insights-agent/README.md [hunk 1]
```diff
@@ -40,6 +40,25 @@ There are several different report types which can be enabled and configured:
...
 See below for configuration details.
 
+## Fleet Installation
+If you're installing the Insights Agent across a large fleet of clusters,
+it can be tedious to use the UI to create each cluster, then copy out the
+cluster's access token.
+
+To better serve customers with a large number of clusters, we've created a flow
+that allows you to easily deploy the Insights Agent across your fleet. You'll
+simply need to set the following flags:
```
Comment (makoscafee) [inline stable/insights-agent/README.md:50]: you probably need to add `insights.organization` is required here too.

File: stable/insights-agent/README.md [hunk 2]
```diff
@@ -40,6 +40,25 @@ There are several different report types which can be enabled and configured:
...
+* `fleetInstall=true` - enable this flow
+* `insights.adminToken=xyz` - you can get this admin token from your organization's settings page at insights.fairwinds.com
+* `insights.tokenSecretName` - the name of the secret where Insights will store your cluster's token. We recommend `insights-token`
+* `insights.cluster` - the name you want to give this cluster in the Insights UI. You might want to auto-generate this from your kubectl context
+
+With these flags set, the Helm chart will create a new cluster in the Insights UI with the specified name
+(unless a cluster with that name already exists) before installing the agent.
+
+When reinstalling the agent, you can omit `adminToken` and `fleetInstall`, and simply specify `tokenSecretName`.
```
Comment (makoscafee) [inline stable/insights-agent/README.md]: reinstalling the agent to the same cluster right?

**Review by makoscafee — APPROVED** [review]
LGTM. just few nits.

File: stable/insights-agent/README.md (hunk 2 above)
Comment (rbren) [reply stable/insights-agent/README.md]: :nod:

File: stable/insights-agent/README.md (hunk 1 above)
Comment (rbren) [reply stable/insights-agent/README.md:50]: 👍 good catch

**Pull request merged**